object-construction-checker/tests/lombok/LombokBuilderExample.java

```diff
@@ -0,0 +1,82 @@
+// Generated by delombok at Wed Aug 14 15:57:15 PDT 2019
```

Comment:
What does this test have to do with the overall PR?

Reply:
Sorry, I forgot to document that. I wanted a testcase with two different required methods that weren't being called so that I could make sure I got the parsing right, and didn't have one. I figured I ought to commit it, even if it was mostly duplicative; we can remove it if we want. Other than the extra required argument it's nearly identical to tests we already have.